include/picongpu/particles/ionization/byField/ADK/ADK.def

```diff
@@ -31,12 +32,13 @@ namespace ionization
     /** Ammosov-Delone-Krainov tunneling model
      *
      * \tparam T_DestSpecies electron species to be created
+     * \tparam T_JIonizationCurrent select type of ionization current (None or EnergyConservation)
```

Comment:
(Not important) do we need `J` as part of name here? As `current` in the name already suggests it's `J`.

Reply:
Is `select` need here? Seems like it's just a type and a called of this code selects all template parameter anyways.

Reply:
Both naming are used in many places, and consistency is more important than a particular choice. So in case you change, please change everywhere. (So it's maybe easier to leave as is everywhere)